doc/classes/SpriteBase3D.xml

```diff
@@ -164,36 +166,51 @@
 		<member name="alpha_cut" type="int" setter="set_alpha_cut_mode" getter="get_alpha_cut_mode" enum="SpriteBase3D.AlphaCutMode">
 		</member>
 		<member name="axis" type="int" setter="set_axis" getter="get_axis" enum="Vector3.Axis">
+			The Direction in wich the front of the texture faces.
 		</member>
 		<member name="centered" type="bool" setter="set_centered" getter="is_centered">
+			If [code]true[/code], texture will be centered. Default value: [code]true[/code].
```

Comment:
The comma here is not necessary. For the rest of the docs we've standardized on no comma after "If true", so please change to follow suit.